tests/system/feature_store/test_feature_store.py

```
@@ -193,9 +195,84 @@ def test_ingest_and_query(self):
+        stocks_set = fs.FeatureSet(
+            "stocks_parquet_test",
+            "stocks set",
+            [Entity("ticker", ValueType.STRING)],
+            timestamp_key="time",
+        )
+
+        df = fs.ingest(stocks_set, stocks_for_parquet, targets)
+        assert len(df) == len(stocks_for_parquet), "dataframe size doesnt match"
+        # assert stocks_set.status.stats["exchange"], "stats not created"
+
+        # test get offline features with different parameters
+        vector = fs.FeatureVector("offline-vec", ["stocks_parquet_test.*"])
+
+        # with_indexes = False, entity_timestamp_column = None
+        default_df = fs.get_offline_features(vector).to_dataframe()
+        assert isinstance(
+            default_df.index, pd.core.indexes.range.RangeIndex
+        ), "index column is not of default type"
+        assert default_df.index.name is None, "index column is not of default type"
+        assert "time" not in default_df.columns, "'time' column shouldn't be present"
+        assert (
+            "ticker" not in default_df.columns
+        ), "'ticker' column shouldn't be present"
+
+        # with_indexes = False, entity_timestamp_column = "time"
+        df_no_time = fs.get_offline_features(
+            vector, entity_timestamp_column="time"
+        ).to_dataframe()
+        assert isinstance(
+            df_no_time.index, pd.core.indexes.range.RangeIndex
+        ), "index column is not of default type"
+        assert df_no_time.index.name is None, "index column is not of default type"
+        assert "time" not in df_no_time.columns, "'time' column shouldn't be present"
+        assert (
+            "ticker" not in df_no_time.columns
+        ), "'ticker' column shouldn't be present"
+
+        # with_indexes = False, entity_timestamp_column = "invalid" - should return the timestamp column
+        df_with_time = fs.get_offline_features(
+            vector, entity_timestamp_column="invalid"
+        ).to_dataframe()
```

should we throw an error? or maybe print a warning in this case? i don't remember what we discussed with dina...